strongbox-security/strongbox-user-management/src/main/resources/etc/conf/strongbox-security-users.yaml

```diff
@@ -5,16 +5,6 @@ users:
       roles:
         - ADMIN
       securityTokenKey: admin-secret
-    - username: maven
-      password: $2a$10$WqtVx7Iio0cndyR1lEaKW.SWhUYmF/zHHG5hkAXvH5hUmklM7QfMO
-      roles:
-        - ADMIN
-      securityTokenKey: maven-secret
-    - username: user
-      password: $2a$10$WqtVx7Iio0cndyR1lEaKW.SWhUYmF/zHHG5hkAXvH5hUmklM7QfMO
-      roles:
-        - ADMIN
-      securityTokenKey: user-secret
     - username: deployer
```

Comment:
I've tested this locally for another branch that I'm working on and the correct settings for the `deployer` user are:
```
    - username: deployer
      password: $2a$10$WqtVx7Iio0cndyR1lEaKW.SWhUYmF/zHHG5hkAXvH5hUmklM7QfMO
      roles:
        - UI_MANAGER
        - ARTIFACTS_MANAGER
      securityTokenKey: deployer-secret
      accessModel:
        storages:
          - id: storage0
            repositories:
              - id: releases
                repositoryPrivileges:
                  - name: ARTIFACTS_DEPLOY
                  - name: ARTIFACTS_RESOLVE
```

Would you mind changing this here as well? Thanks! :)

Reply:
Sorry, I had a bit of a mistake and corrected it. It should be just the above now.

Reply:
ok, updating! :)

Reply:
Thanks and sorry about this!